web/src/component/community/Ranking/Amount/RestRanking.tsx

```diff
@@ -0,0 +1,22 @@
+interface RestRankingProps {
```

Comment:
그리고 RestRankingProps랑 TopThreeProps랑 같은 인터페이스인데, 둘이 따로 나누고 싶으면

`BaseProps` 이런거 만들어서 공통으로 묶은다음에 각 Props에 extend 하는 방식으로 하면 좋을거 같은데
너의 생각은 어떠니 이렇게 만든 이유가 있을까?

```suggestion
interface RestRankingProps extends BaseProps {
```

Reply:
각 파일에 각각 만들어야 한다고 밖에 생각을 못했던 것 같아. `BaseProps` 만들어서 한 번 적용해볼게!